src/components/Steps/__stories__/steps.stories.mdx

```diff
@@ -162,8 +209,18 @@ Our Tipseen component includes support for steps as content.
               <TipseenWizard
                 animationType={Tipseen.animationTypes.OPACITY_AND_SLIDE}
                 title="This is a title"
-                steps={content}
-                activeStepIndex={2}
+                steps={steps5WithText}
```

Comment:
the reason we did not used the global steps for the examples was for the readers be able to see how the tipseen content should look like